src/startDebugging.ts

```diff
@@ -72,7 +72,7 @@ function getDebugLaunchConfig(folder: vscode.WorkspaceFolder, workflowID: string
     name: `Time-Travel Debug ${workflowID}`,
     type: 'node-terminal',
     request: 'launch',
-    command: `npx dbos-sdk debug -x http://localhost:${proxyPort} -u ${workflowID}`,
+    command: `npx dbos-sdk debug -x postgresql://127.0.0.1:${proxyPort} -u ${workflowID}`,
```

Comment:
should we drop `dbos-sdk` in favor of just `dbos` while we're at this?

Reply:
Done.